components/QuestionControl.tsx

```diff
@@ -45,9 +54,20 @@ export const QuestionControl = memo<{
 								: "text"
 						}
 						{...(definition.type === "number" ? { min: 0 } : {})}
+						maxLength={
+							Postcode.indexOf(definition.question.toLowerCase()) > -1 ? 5 : 999
```

Comment:
misto 999 bych dal undefined, at to neni zbytecne omezeny

Reply:
`Postcode.indexOf(definition.question.toLowerCase())` by chtelo abstrahovat ven

Reply:
`isPostcode` methoda  toho /lib/shared